modules/ROOT/pages/clauses/load-csv.adoc

```diff
@@ -158,6 +158,46 @@ Added 4 nodes, Set 8 properties, Added 4 labels
 |===
 ====
 
+[role=label--enterprise-edition label--not-on-aura label--new-5.21]
```

Comment:
are the labels correct? is it enterprise edition? what about aura?

Reply:
Correct - enterprise only and not on Aura